Tools/Scripts/webkitpy/benchmark_runner/webserver_benchmark_runner.py

```diff
@@ -63,7 +63,7 @@ def _run_one_test(self, web_root, test_file, iteration):
         except Exception as error:
             self._browser_driver.diagnose_test_failure(self._diagnose_dir, error)
             raise error
-        finally:
+        else:
             self._browser_driver.close_browsers()
             self._http_server_driver.kill_server()
```

Comment:
Server is running with Popen, which is a child process of current process, if the current process exits, child process will exit. So the server will not be running on exception anyway. I think it would be cleaner if we explicitly kill it.

Reply:
I updated the code to always kill the web server.